corehq/apps/domain/models.py

```diff
@@ -635,7 +635,7 @@ def get_all(cls, include_docs=True):
         if not include_docs:
             return domains
         else:
-            return imap(cls.wrap, iter_docs(cls.get_db(), [d['id'] for d in domains]))
+            return map(cls.wrap, iter_docs(cls.get_db(), [d['id'] for d in domains]))
```

Comment:
Will pylint/stickler catch future uses of map?

Reply:
I *think* it's smart enough to tell the difference between the builtin map and an imported map